cpp/src/barretenberg/proof_system/circuit_constructors/ultra_circuit_constructor.cpp

```diff
@@ -100,23 +100,17 @@ void UltraCircuitConstructor::add_gates_to_ensure_all_polys_are_non_zero()
     // to get a non-zero value in table_4. I assume this index is arbitrary and could
     // start from 1 instead of 0?
     uint32_t left_value = 3;
-    uint32_t right_value = 5;
+    uint32_t right_value = 3;
```

Comment:
can you explain what this value is because the change looks arbitrary

Reply:
The change is because the multitable only works on 2-bit values, so 5 is too big